src/utils/riskUtils.js

```diff
@@ -1,10 +1,12 @@
 import handlebars from 'handlebars'
 import RecaptchaV2Invisible from "./risk-providers/recaptcha-v2-invisible";
 import RecaptchaV3 from "./risk-providers/recaptcha-v3";
+import RecaptchaEnterprise from "./risk-providers/recaptcha-enterprise";
 import Signals from "./risk-providers/signals";
 
 const CAPTCHA_TYPE_RECAPTCHA_V2_INVISIBLE = "reCAPTCHA v2 Invisible";
 const CAPTCHA_TYPE_RECAPTHCA_V3 = "reCAPTCHA v3";
```

Comment:
Not your change, but do you mind fixing the typo for in the variable name for RECAPTCHA v3? Thanks.

Reply:
Sure, fixed it with the latest commit.